gef-ui/server/controllers/amendments/amendment-details/create-amendment-details-view-model.ts

```diff
@@ -37,7 +37,7 @@ export const createAmendmentDetailsViewModel = ({
   amendmentId: amendment.amendmentId,
   effectiveDate: amendment.effectiveDate ? format(fromUnixTime(amendment.effectiveDate), DATE_FORMATS.D_MMMM_YYYY) : '',
   banner,
-  canAbandonFacilityAmendment: userRoles.includes('maker') && amendment.status === PORTAL_AMENDMENT_STATUS.FURTHER_MAKERS_INPUT_REQUIRED,
+  canSubmitFacilityAmendment: userRoles.includes('maker') && amendment.status === PORTAL_AMENDMENT_STATUS.FURTHER_MAKERS_INPUT_REQUIRED,
```

Comment:
const canSubmitFacilityAmendment = userRoles.includes('maker') && amendment.status === PORTAL_AMENDMENT_STATUS.FURTHER_MAKERS_INPUT_REQUIRED;

```suggestion
  canSubmitFacilityAmendment,
```

* Can you please check whether there is already a helper function for the same?
* Please check `maker` from constant.

Reply:
I can't see any helper to do this